concent-vm/Vagrantfile

```diff
@@ -12,7 +12,8 @@ Vagrant.configure("2") do |config|
 
     config.vm.network "private_network", ip: "172.40.2.3"
 
-    golem_configuration_directory = './disk/golem_configuration_directory.vdi'
+    # Path must be absolute to work correctly on Ubuntu with Vagrant 2.2.1.
+    golem_configuration_directory = File.expand_path('./disk/golem_configuration_directory.vdi')
```

Comment:
I'd rephrase this comment because now it sounds like "it doesn't work on Ubuntu with Vagrant 2.2.1 but I won't tell you why" :)
``` python
# This should be an absolute path because `vboxmanage` fails when used with
# a relative one on some systems (discovered on Ubuntu xx.xx with Vagrant 2.2.1).
```

Reply:
@rwrzesien What about my remark here?

Reply:
I think I must have missed it, should I add it now?

Reply:
I have already added that change in #325. You could review it :)